main/src/main/java/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1932,20 +1950,22 @@ static boolean editModifiedCoordinates(@NonNull final Geocache cache, final Geop
 
         final String uriPrefix = "https://www.geocaching.com/seek/cache_details.aspx/";
 
-        try {
-            Network.completeWithSuccess(Network.postJsonRequest(uriPrefix + uriSuffix, jo));
-            Log.i("GCParser.editModifiedCoordinates - edited on GC.com");
-            return true;
-        } catch (final Exception ignored) {
-            Log.e("GCParser.deleteModifiedCoordinates - cannot delete modified coords");
-            return false;
-        }
+        return Network.completeWithSuccess(Network.postJsonRequest(uriPrefix + uriSuffix, jo))
+            .toSingle(() -> {
+                Log.i("GCParser.editModifiedCoordinates - edited on GC.com");
+                return true;
+            })
+            .onErrorReturn((throwable) -> {
+                Log.e("GCParser.deleteModifiedCoordinates - cannot delete modified coords", throwable);
+                return false;
```

Comment:
On other places you changed to `return Single.just(false);` (or `true`).
Should here also, isn´t it?

Reply:
on Error Return expects a value rather than Single of value